### ISSUE TITLE
Different behavior of redirection urls?

### ISSUE THREAD

**Issue by darkyen** [ForbesLindesay/brcdn.org#8]
I am using this to fetch dependencies from a package from a git module (after a lot of magic) to generate a runnable module with custom conditions. (For a POC only). Is it possible to directly fetch result url ?

Aka https://www.brcdn.org/?module=throat&version=2.0.2 redirecting me straight to //brcdn.org/qWQsQgoIKxqLm7k1j5WHfhACgw.js

**Comment by darkyen** [comment]
Sorry, I was using https://www.brcdn.org/?module=throat&version=2.0.2 should use https://www.brcdn.org/throat/2.0.2?
Confusing !

**Comment by ForbesLindesay** [comment]
Yes, this is sort of touched on, but not very clearly, in the home page.
